cmd/backup-manager/app/export/manager.go

```diff
@@ -258,17 +258,29 @@ func (bm *BackupManager) performBackup(ctx context.Context, backup *v1alpha1.Bac
 		return err
 	}
 
-	backupErr := bm.dumpTidbClusterData(ctx, backup)
+	backupErr := bm.dumpTidbClusterData(ctx, backupFullPath, backup)
+	if backupErr != nil {
+		errs = append(errs, backupErr)
+		klog.Errorf("dump cluster %s data failed, err: %s", bm, backupErr)
+		uerr := bm.StatusUpdater.Update(backup, &v1alpha1.BackupCondition{
+			Type:    v1alpha1.BackupFailed,
+			Status:  corev1.ConditionTrue,
+			Reason:  "DumpTidbClusterFailed",
+			Message: backupErr.Error(),
+		}, nil)
+		errs = append(errs, uerr)
+		return errorutils.NewAggregate(errs)
```

Comment:
why move this to the above of `if oldTikvGCTimeDuration < tikvGCTimeDuration`?

Reply:
backupErr should be checked immediately after being returned

Reply:
@liubog2008 We should not move this as we have to change back the GC time no matter the backup succeeds or not.

Reply:
OK...but what happened if job is crashed before we change the GC time back?